app/shared/helpers/dates.js

```diff
@@ -13,6 +13,19 @@ dateToCalendar = function (date) {
 	return moment(date).calendar();  
 };
 
+millisecondsToDate = function(milliseconds) {
+	if ( !(milliseconds) ) {
```

Comment:
(!(something)) really hurts the readability argument of !(). Change to == false or == null?